charmcraft.yaml

```diff
@@ -164,6 +167,7 @@ parts:
   charm:
     build-packages:
     - git
+    charm-binary-python-packages: [importlib-metadata]
```

Comment:
Same old friend that keeps haunting us 😄

Reply:
👻 importlib metadata 👻